app/__init__.py

```diff
@@ -1,35 +1,33 @@
 from flask import Flask
-from flask_login import LoginManager,login_required
+from flask_login import LoginManager, login_required
 from flask_sqlalchemy import SQLAlchemy
-app=Flask(__name__)
+from flask_migrate import Migrate
+app =Flask(__name__)
```

Comment:
You are initiation` FLASK(__name__)` twice. Why not remove this and use the one on `line 10`